docs/handbook/01_intro.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+Quantum computing has the potential to revolutionize many fields in the 21st century. Over the past decade, numerous quantum computers from multiple providers based on different qubit technologies have been made publicly available.
```

Comment:
Should we maybe cite some references on potential industrial applications?

Reply:
Done in https://github.com/cda-tum/mqt/pull/10/commits/0f8a7402fe827d22aadaf7da4975c13205fe4a4c 👍🏼